### PR TITLE
sys-cluster/drbd-utils: Bump to 9.15.1

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	# bug 616758
+	#epatch "${FILESDIR}/${P}"-sysmacros.patch
+
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir=/var \
```
Comment (SoapGentoo) [inline sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild:59]: `EPREFIX`

File: sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -457,8 +457,8 @@ AH_TEMPLATE(DRBD_LEGACY_84, [Include support for drbd-8.4 kernel code])
+ AH_TEMPLATE(WINDRBD, [Include support for windrbd (do not use this under Linux)])
+ 
+ AC_SUBST(DRBD_LIB_DIR, [$localstatedir/lib/$PACKAGE_TARNAME])
+-AC_SUBST(DRBD_RUN_DIR, [$localstatedir/run/$PACKAGE_TARNAME])
+-AC_SUBST(DRBD_LOCK_DIR, [$localstatedir/lock])
++AC_SUBST(DRBD_RUN_DIR, [/run/$PACKAGE_TARNAME])
++AC_SUBST(DRBD_LOCK_DIR, [/run/lock])
```
Comment (SoapGentoo) [inline sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch:10]: I guess Autoconf 2.70 would solve this

Comment (jsmolic) [reply sys-cluster/drbd-utils/files/drbd-utils-9.15.1-run-lock.patch:10]: Okay but I'll leave the patch until it's unmasked

File: sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild (hunk 1 above)
Comment (jsmolic) [reply sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild]: Done

File: sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+	newinitd "${FILESDIR}"/${PN}-8.0.rc ${PN/-utils/}
+
+	dodoc scripts/drbd.conf.example
+
+	keepdir /var/lib/drbd
+	rm -r "${ED}"/var/run || die
+
+	# https://bugs.gentoo.org/698304
+	dodir lib/drbd
+	for i in drbd drbdadm-83 drbdadm-84 drbdsetup-83 drbdsetup-84; do
```
Comment (SoapGentoo) [inline sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild:86]: `local i`

File: sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+	sed -i -e '/usage-count/ s/yes/no/' scripts/global_common.conf || die
+	sed -i -e "s:\$(sysconfdir)/udev:$(get_udevdir):" scripts/Makefile.in || die
+
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir="${EPREFIX}"/var \
```
Comment (SoapGentoo) [inline sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild:57]: can this actually work with the current non-masked autoconf? :thinking:

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild (hunk 3 above)
Comment (jsmolic) [reply sys-cluster/drbd-utils/drbd-utils-9.15.1.ebuild:87]: Done

**Comment by jsmolic** [comment]
Closing in favour of https://github.com/gentoo/gentoo/pull/19481

**Pull request closed without merging**